src/utils/image.js

```diff
@@ -2,7 +2,12 @@ import fetch from 'isomorphic-fetch';
 import PNG from '@react-pdf/png-js';
 import JPEG from './jpeg';
 
-const Buffer = require('buffer/').Buffer;
+const isValidFormat = format => {
+  const lower = format.toLowerCase();
+  return lower === 'jpg' || lower === 'jpeg' || lower === 'png';
+};
+
+const isBase64 = src => /data:image\/[a-zA-Z]*;base64,[^\"]*/g.test(src);
```

Comment:
I'm no expert on this, but reading up on matching for base64 it does seem a bit more complex than this suggests. I understand that separating data and filetype is the objective here, so it makes sense to keep it simple. I would suggest renaming the function to `isCompatibleBase64`, just to make this distinction clear.